platforms/core-execution/persistent-cache/src/main/java/org/gradle/cache/internal/FixedExclusiveModeCrossProcessCacheAccess.java

```diff
@@ -57,12 +56,14 @@ public FixedExclusiveModeCrossProcessCacheAccess(String cacheDisplayName, File l
         this.lockManager = lockManager;
     }
 
+    private static void noOpContentAction(FileLockReleasedSignal unused) {}
+
     @Override
     public void open() {
         if (fileLock != null) {
             throw new IllegalStateException("File lock " + lockTarget + " is already open.");
         }
-        final FileLock fileLock = lockManager.lock(lockTarget, lockOptions, cacheDisplayName, "", NO_OP_CONTENDED_ACTION);
+        final FileLock fileLock = lockManager.lock(lockTarget, lockOptions, cacheDisplayName, "", FixedExclusiveModeCrossProcessCacheAccess::noOpContentAction);
```

Comment:
💅 same as platforms/core-execution/persistent-cache/src/main/java/org/gradle/cache/internal/DefaultCacheCoordinator.java, an inlined lambda would be clearer.